src/smdba/postgresqlgate.py

```diff
@@ -155,9 +155,7 @@ def estimate(self):
         # No more than 1GB
         self.config['maintenance_work_mem'] = self.to_mb(self.br((mem / 0x10) > megabytes and megabytes or mem / 0x10))
 
-        self.config['checkpoint_segments'] = 8
```

Comment:
I understand this removal for 9.6, but I do not understand completely missing [max/min WAL size values](https://www.postgresql.org/docs/9.6/static/runtime-config-wal.html#GUC-MIN-WAL-SIZE)...